sequence_processing_pipeline/NuQCJob.py

```diff
@@ -414,13 +416,10 @@ def _generate_job_script(self, max_bucket_size):
         # a string.
         if max_bucket_size < gigabyte:
```

Comment:
@antgonza Do we still want conditionals for wall_time_limit and gres_value (see above)?

Reply:
Fine to delete and also fine to remove the "extra lines" for `mod_wall_time_limit `; in other words, no need to modify `mod_wall_time_limit ` and `mod_jmem`; thanks.